### PR TITLE
refactor: Converted the PerformanceHome to function component.

### PR DESCRIPTION
**Pull request by Eezi**

### CODE REVIEW

File: packages/client/src/views/SysAdmin/Team/user/userCreation/UserReviewForm.tsx [hunk 1]
```diff
@@ -274,100 +275,89 @@ class UserReviewFormComponent extends React.Component<
...
+        }
+        onClick={handleSubmit}
+        icon={() => <Check />}
+        align={ICON_ALIGNMENT.LEFT}
+      >
+        {intl.formatMessage(buttonMessages.confirm)}
+      </SuccessButton>
+    )
+  } else {
+    title = section.title && intl.formatMessage(section.title)
```
Comment (Siyasanga) [inline packages/client/src/views/SysAdmin/Team/user/userCreation/UserReviewForm.tsx:306]: Hi @Eezi it seems like `section` is undefined here, you may have have forgoten to destructure it from `props`

**Review by Siyasanga — COMMENTED** [review]
Hi @Eezi thanks for the changes, everything looks good except for one error I saw in the UserReviewForm. I left a comment on it

**Comment by Eezi** [comment]
Thanks for reviewing the code!

**Review by Siyasanga — APPROVED** [review]
Awesome, LGTM!

**Comment by Siyasanga** [comment]
@Eezi I'm happy with the changes, please feel free to merge using the `squash and merge` strategy when you are ready

**Comment by Siyasanga** [comment]
@rikukissa do you if the changes from this PR are going to go into a specific milestone?

**Comment by rikukissa** [comment]
@Siyasanga No, just directly to develop is fine 👍

**Pull request merged**